db/migrations/20220308103000_replace_check_balance.up.go

```diff
@@ -0,0 +1,86 @@
+package migrations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/uptrace/bun"
+)
+
+func init() {
+	Migrations.MustRegister(func(ctx context.Context, db *bun.DB) error {
+
+		if db.Dialect().Name().String() != "pg" {
+			fmt.Printf("\033[1;31m%s\033[0m", "You are not using PostgreSQL. DB level checks can not be enabled!\n")
+			return nil
+		}
+		sql := `
+			-- make sure that account balances >= 0 (except for incoming and fees accounts)
+				CREATE OR REPLACE FUNCTION check_balance()
```

Comment:
create or replace should re-create check_balance function so i just copied this one with fees balance constraint